tests/foreman/cli/test_template.py

```diff
@@ -53,27 +46,21 @@ def test_update_template_1(self):
         @Assert: Template is updated
 
         """
-
-        content = gen_string("alpha", 10)
         name = gen_string("alpha", 10)
 
         try:
-            new_obj = make_template(
-                {
-                    'name': name,
-                    'content': content,
-                }
-            )
+            new_obj = make_template({
+                'name': name,
+                'content': gen_string("alpha", 10),
+            })
         except CLIFactoryError as e:
             self.fail(e)
 
-        result = Template.info({'id': new_obj['id']})
+        updated_name = gen_string("alpha", 10)
```

Comment:
`s/"/'/g`
`s/, 10//`? (Signature of [`gen_alpha`](https://github.com/omaciel/fauxfactory/blob/master/fauxfactory/__init__.py#L147))